ci-operator/step-registry/openstack/ovsdpdk/deploypao/openstack-ovsdpdk-deploypao-commands.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
```

Comment:
I think we don't have to do anything but his will have to be checked, if KUBECONFIG is loaded for us already.